src/components/Cover/index.stories.js

```diff
@@ -24,24 +25,18 @@ export default {
   title: 'Cover',
 }
 
-export const basic = () => (
-  <Cover images={data.randomBackgroundImage.imageSizes}>
-    <H1>Awakenings Festival 2019</H1>
-  </Cover>
-)
+const StyledCover = styled(Cover)`
+  text-align: center;
+`
```

Comment:
I would not style the storybook, that will make it harder to see the intentional design and usage.